app/src/main/kotlin/Main.kt

```diff
@@ -1,8 +1,7 @@
 package me.john_
 
-import ConnectFourGame
+import Reversi.*
 
```

Comment:
@JunTaoLuo would it be better to make a separate main for each game?

Reply:
Yea, I'll rebase.